pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1379,18 +1378,19 @@ func SendMetricsProtobuf(ReportMetrics *metrics.ZMetricMsg,
 // Use the ifname/vifname to find the underlay status
 // and from there the (ip, allocated, mac) addresses for the app
```

Comment:
Add "Returns one IPv4 address and a list of IPv6 addresses

Reply:
done